BUILDING.md

```diff
@@ -603,10 +603,12 @@ platforms to avoid issues with Boost config files (introduced in Boost version
 to use Boost specified config files for their USD build, specify 
 -DBoost_NO_BOOST_CMAKE=OFF when running cmake.
 
-2. Windows and Python 3.8+
+2. Windows and Python 3.8+ (non-Anaconda)
 Python 3.8 and later on Windows will no longer search PATH for DLL dependencies.
 Instead, clients can call `os.add_dll_directory(p)` to set paths to search.
 By default on that platform USD will iterate over PATH and add all paths using
 `os.add_dll_directory()` when importing Python modules. Users may override
 this by setting the environment variable `PXR_USD_WINDOWS_DLL_PATH` to a PATH-like
-string. If this is set, USD will use these paths instead.
+string. If this is set, USD will use these paths instead.  (Note that Anaconda
+python interpreters are modified to behave like pre-3.8 python interpreters,
+and so they continue to use the PATH.)
```

Comment:
I don't think this comment is accurate, with this change it will be true for the pypi packages but not for builds in general. If we want to make this something that affects all builds, we could move the new logic into `Tf/__init__.py`.

Reply:
Talked to @pmolodo out of band, and my note about changing it in Tf is not right. Basically anything we do there won't have any effect under conda unless CONDA_DLL_SEARCH_MODIFICATION_ENABLE is set. I think what he was trying to say here is that conda just doesn't do the new thing so users need to configure the old way, but I misunderstood.

Reply:
I pushed a new version that hopefully clarifies the wording.  Thanks for pointing this out as potentially confusing!